pkg/collector/corechecks/snmp/internal/report/report_utils.go

```diff
@@ -170,3 +170,26 @@ func getInterfaceConfig(interfaceConfigs []snmpintegration.InterfaceConfig, inde
 	}
 	return snmpintegration.InterfaceConfig{}, fmt.Errorf("no matching interface found for index=%s, tags=%s", index, tags)
 }
+
+// getConstantMetricValues retrieve all metric tags indexes and set their value as 1
+func getConstantMetricValues(mtcl checkconfig.MetricTagConfigList, values *valuestore.ResultValueStore) map[string]valuestore.ResultValue {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you add a unit test for this function? 

Small functions like this one should be ideally unit-tested, and ideally for all code paths (happy path and error paths e.g. `error getting column value`).

Reply:
Of course thank you!  Test got lost in my local history 🤦‍♂️